modules/db/src/main/Handlers.scala

```diff
@@ -108,18 +111,18 @@ trait Handlers:
       leftHandler.readTry(bson).map(Left.apply).orElse(rightHandler.readTry(bson).map(Right.apply))
     def writeTry(e: Either[L, R]) = e.fold(leftHandler.writeTry, rightHandler.writeTry)
 
-  def stringMapHandler[V](using
```

Comment:
reactivemongo provides `mapWriter/Reader` and `collectionWriter/Reader` which are both valid instance for `Map[String, V]`, and now it is illegal in 3.6.
With new implement of `mapHandler` We use `mapWriter` and `mapReader` directly which avoid the issue.